modules/local/gzip_files.nf

```diff
@@ -0,0 +1,31 @@
+/*
+GZIP output files
+*/
+
+process GZIP_FILES {
+    tag "${meta.id}"
+    label 'process_low'
+    container "${workflow.containerEngine == 'singularity' || workflow.containerEngine == 'apptainer' ? task.ext.containers.get('singularity') : task.ext.containers.get('docker')}"
+
+    input:
+    tuple val(meta), path(file_in)
+
+
+    output:
+    tuple val(meta), path(output_name), emit: zipped_file
+    path "versions.yml", emit: versions
+
+
+    script:
+    output = file_in.getName()
+    output_name = "${output}.gz"
+    """
+    gzip -c ${file_in} > ${output_name}
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        gzip: \$(echo \$(touch --version 2>&1) | sed 's/^.*coreutils) //; s/ .*\$//')
```

Comment:
You are taking the version from `touch`. Could you get the version from `gzip --version` instead?

Reply:
fixed in: adeecd7cc6f6f7ee8c270f5ac5e69899f1bb7d5e